en/README.md

```diff
@@ -0,0 +1,1138 @@
+
+***params***
+
+params content is same as the result object of [getMonitoringServerInfo](#user-content-getmonitoringserverinfomethod).
+
+```json
+{
+  "method": "updateMonitoringServerInfo",
+  "params": {
+    "extendedInfo": "exampleExtraInfo",
+    "serverId": 1,
+    "url": "http://example.com:80",
+    "type": "12345678-9abc-def0-1234-567891abcdef",
+    "nickName": "exampleName",
+    "userName": "Admin",
+    "password": "examplePass",
+    "pollingIntervalSec": 30,
+    "retryIntervalSec": 10
+  },
+  "jsonrpc": "2.0"
+}
+```
+
+## Tables
+
+### armInfoStatus
+
+|Status|Description|
+|:---------|:---|
+|"INIT"   |Initial status. Not communication yes.|
+|"OK"     |Succeed connection.|
+|"NG"     |Fail connection.|
+
+### ServerType
+
+If you create a HAP, need to define new server type that has UUID other than the following UUIDs.
+
+|$BL>A0(B|UUID|
+|:---|:---|
+|Zabbix    |8e632c14-d1f7-11e4-8350-d43d7e3146fb|
+|Nagios    |902d955c-d1f7-11e4-80f9-d43d7e3146fb|
+|Ceilometer|aa25a332-d1f7-11e4-80b4-d43d7e3146fb|
+
+### triggerSeverity
+
+|Severity|Description|
+|:---|:--|
+|"UNKNOWN"  |Unknown|
+|"INFO"     |Information|
+|"WARNING"  |Warning|
+|"ERROR"    |Error|
+|"CRITICAL" |Critical|
+|"EMERGENCY"|Emergency|
+
+### triggerStatus
+
+|Status|Description|
+|:---------|:---|
+|"OK""    |Succeed connection.|
+|"NG"     |Fail connection.|
+|"UNKNOWN"|Status is unknown|
+
+### updateType
+
+|Type|Description|
+|:---------|:---|
+|"ALL"    |Send all data. Remove old data from hatohol server and regist all sent data.|
+|"UPDATED"|Send only updata data, after that If data ID already exists in Hatohol server, overwrite data and in case of does not exist, initial regist that data.|
+
+### eventType
+
+|Type|Description|
+|:---|:---------:|
+|"GOOD"        |Normal|
+|"BAD"         |Abnormal|
+|"UNKNOWN"     |Unknown|
+|"NOTIFICATION"|Notification|
+
+### putResult
+
+The following table value are success and failure of to call put procedure. If fail to update database, to recall procedure is standard motion.
+
+|Status|Description|
+|:---------|:---|
+|"SUCCESS"|Update is succeed.|
+|"FAILURE" |Update is fail.|
+
+ In case of put procedure params wronged and etc..., need to return error that is defined in the JSON_RPC2.0 to HAP as the following example.
+
+ ```json
+{
+  "id": "1",
+  "error": {
+    "message": "Invalid params",
+    "code": -32602
+  },
+  "jsonrpc": "2.0"
+}
+ ```
+
+### fetchResult
+
+The following table value are success and failure of fetch request. If request fail, to resend request to plugin is standard motion.
+
+|Status|Description|
+|:---------|:---|
+|"SUCCESS"|Succeed to accept a request.|
+|"ABBREV" |Abbreviate to accept a request because request interval a little.| 
+|"FAILURE"|Fail to accept a request.|
+
+ In case of fetch procedure params wronged and etc..., need to return error that is defined in the JSON_RPC2.0 to Hatohol server as the following example.
```

Comment:
> In case of fetch procedure params wronged and etc..., need to return error that is defined in the JSON_RPC2.0 to Hatohol server as the following example.

If fetch series of procedures' parameters are wrong or invalid and so on, it needs to return error which is defined in the JSON RPC 2.0 to HAP as following example: